ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -285,24 +293,25 @@ export default React.memo<Props>(function VideoJs(props: Props) {
     if (e.keyCode === SEEK_FORWARD_KEYCODE) {
       const newDuration = currentTime + SEEK_STEP;
       videoNode.currentTime = newDuration > duration ? duration : newDuration;
+      OVERLAY.showSeekedOverlay(player, SEEK_STEP, true);
     }
     if (e.keyCode === SEEK_BACKWARD_KEYCODE) {
       const newDuration = currentTime - SEEK_STEP;
       videoNode.currentTime = newDuration < 0 ? 0 : newDuration;
+      OVERLAY.showSeekedOverlay(player, SEEK_STEP, false);
     }
 
     // Playback-Rate Shortcuts ('>' = speed up, '<' = speed down)
     if (player && e.shiftKey && (e.keyCode === PERIOD_KEYCODE || e.keyCode === COMMA_KEYCODE)) {
+      const isSpeedUp = e.keyCode === PERIOD_KEYCODE;
       const rate = player.playbackRate();
       let rateIndex = videoPlaybackRates.findIndex(x => x === rate);
       if (rateIndex >= 0) {
-        rateIndex =
-          e.keyCode === PERIOD_KEYCODE
-            ? Math.min(rateIndex + 1, videoPlaybackRates.length - 1)
-            : Math.max(rateIndex - 1, 0);
+        rateIndex = isSpeedUp ? Math.min(rateIndex + 1, videoPlaybackRates.length - 1) : Math.max(rateIndex - 1, 0);
+        const nextRate = videoPlaybackRates[rateIndex];
 
-        player.userActive(true); // Bring up the ControlBar as GUI feedback.
```

Comment:
I think it's still helpful to bring up the control bar when someone skips forward